doc-content/enterprise-only/getting-started/decision-table-penalty-columns-proc.adoc

```diff
@@ -0,0 +1,26 @@
+[id='penalty-columns-proc']
+= Inserting `Fine Amount` and `Points` columns
+
+The `Fine Amount` and `Points` columns contain the fines and points based on the corresponding `Violation Type` field values.
+
+.Prerequisite
+
+Inserted the `Violation Type` column.
```

Comment:
You have inserted. I actually think this doesn't give enough context. Maybe something like 'You have inserted the Violation Type column in (where ever), in Decision Central. The idea is the prerequisites should give enough for the reader get to the starting point for the first step, assuming they haven't read the rest of the doc (but would know how to get to this point). Might be worth reviewing all of your prereqs from this POV.

Reply:
Corrected. Thank you.